perceval/backends/core/github.py

```diff
@@ -155,6 +157,9 @@ def fetch(self, category=CATEGORY_ISSUE, from_date=DEFAULT_DATETIME, to_date=DEF
 
         :returns: a generator of issues
         """
+        if self.exclude_user_data:
+            logger.info("Excluding user data. Won't get user data for the login")
```

Comment:
`Won't get user data for the login` seems too specific, maybe it could be rewritten as:
`Excluding user data. Personal user information won't be collected from the API.`